.ruff.toml

```diff
@@ -410,6 +410,9 @@ select = [
 "tests/test_util/test_util_typing.py" = ["UP006", "UP035"]
 "tests/test_util/typing_test_data.py" = ["FA100", "UP006", "UP035"]
 
+# linkcheck builder tests use the contextmanager variable in subsequent 'with' blocks
+"tests/test_builders/test_build_linkcheck.py" = ["SIM117"]
```

Comment:
I thought this might have been a bug in `ruff` (derived from `flake8-simplify`, where `SIM117` originated).

It's not, though.  It turns out that the pattern is valid Python:

```python
with a() as var, b(var) as result:
    ...
```

(reusing a `with` target/result in a subsequent element)

I'm considering undoing this `.ruff.toml` change and applying the guidance.

Reply:
It's better to have a single `with` with multiple instantiations rather than having nested with (they are equivalent by the way)

Reply:
Ok, thank you.  I'll make that change :+1: